integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java

```diff
@@ -820,7 +820,7 @@ public void shouldBeAbleToWriteAfterCallingReadOnlyProcedure()
     private static List<Exception> exceptionsInProcedure = Collections.<Exception>synchronizedList( new ArrayList<>() );
 
     @Test
-    public void shouldGracefullyFailWhenSpawningThreadsCreatingTransactionInProcedures() throws Throwable
+    public void shouldBeAbleToSpawnThreadsCreatingTransactionInProcedures() throws Throwable
     {
         // given
         Runnable doIt = () -> {
```

Comment:
Perhaps rename the procedure, now that it is actually supported. Perhaps `createNodeInNewTransaction()`?